packages/imperative/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added progress tracking to the `DirToUSSDirHandler` by using `commandParameters.response.progress` to show progress bars during file uploads. [#2334](https://github.com/zowe/zowe-cli/issues/2344)
```

Comment:
we should explain what `DirToUSSDirHandler` and `commandParameters.blah.blah` are: "... to the class/profile/cat/whatever `DirToBlahBlah` by using the `commandBlahBlah` function/class/cat/whatever to ..."